frontend/src/component/admin/adminRoutes.ts

```diff
@@ -90,7 +90,7 @@ export const adminRoutes: INavigationMenuItem[] = [
     {
         path: '/admin/admin-invoices',
         title: 'Billing & invoices',
-        menu: { adminSettings: true, mode: ['pro'], billing: true },
+        menu: { adminSettings: true, billing: true },
```

Comment:
Logic to show menu item is already handled by `useInstanceStatus()`, no need to specify "mode"